src/backend/joanie/core/migrations/0050_alter_contractdefinition_update_name.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.16 on 2024-12-02 17:20
+
+from django.db import migrations, models
+
+def migrate_contract_definition_name(apps, schema_editor):
+    """
+    Updates all existing ContractDefinition records with the name 'contract_definition'
+    to 'contract_definition_unicamp'. This ensures consistency with the new naming convention.
+    """
+    ContractDefinition = apps.get_model("core", "ContractDefinition")
+    ContractDefinition.objects.filter(
+        name="contract_definition"
+    ).update(name="contract_definition_unicamp")
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0049_alter_contractdefinition_name'),
+    ]
+
+    operations = [
+        # Update every existing contract definition objects `name` to be "contract_definition_unicamp"
+        migrations.RunPython(migrate_contract_definition_name, migrations.RunPython.noop),
+    ]
```

Comment:
In a previous comment, when I told about add a rollback function, I think about something like that :
https://github.com/openfun/joanie/blob/ac1026bc8d9186fada3078bd85c125c2848e2c51/src/backend/joanie/payment/migrations/0005_invoice_recipient_address.py#L81-L83